README.md

```diff
@@ -42,21 +46,17 @@ dc exec app python manage.py price
 #### Файлы env
 `make deploy-dev` создаст файлы для окружения (env) со стандартными значениями.
 А затем попросит заполнить их.
+
 Пару рекомендаций по заполнению:
 - Генерим случайные: Django secret key, пароли к локальным базам
+- В файлах `shopelectro/docker/env_files/paths` и `shopelectro/docker/.env` определяем путь `REFARM_DIR=../../refarm-site`. Интерфейс refarm-site нестабилен, поэтому иногда при разработке фичи сайта
```

Comment:
`../../refarm-site` path is the example. Concrete one depends from git clone steps. Mention it in 3-7 words plz